app/models/individualDetails/IndividualDetails.scala

```diff
@@ -223,7 +223,7 @@ object IndividualDetails {
     def getNino: String = getNinoWithoutSuffix + idData.ninoSuffix.get.value
 
     def getFirstForename: String = idData.nameList.name.flatMap(_.headOption).map(_.firstForename.value).getOrElse("")
-    private def getSecondForename: String = idData.nameList.name.flatMap(_.drop(1).headOption).map(_.firstForename.value).getOrElse("")
+    private def getSecondForename: String = idData.nameList.name.flatMap(_.headOption).map(_.secondForename.get.value).getOrElse("")
```

Comment:
Won't `secondForname.get` throw an exception if it is not present in the payload?